File: C/Funzioni/Cai/v2/cai.c
#include "cai.h"
#include "random.h"


void cai_init(){
  random_init();
}

void cai_sbagliato(){
  int num = random_between(0, 4);

  switch(num){
    case 0:
      puts("No. Please try again.");
      break;
    case 1:
      puts("Wrong. Try once more.");
      break;
    case 2:
      puts("Don't give up!");
      break;
    case 3:
      puts("No. Keep trying.");
      break;
  }

}

void cai_giusto(){
  int num = random_between(0, 4);

  switch(num){
    case 0:
      puts("Very Good!");
      break;
    case 1:
      puts("Excellent!");
      break;
    case 2:
      puts("Nice work!");
      break;
    case 3:
      puts("Keep up the good work!");
      break;
  }
}

int cai_estrai_num(){
  return random_between(1, 10);
}

File: Python/Cai/cai.py
import random

def sbagliato():
  num = random.randrange(0, 4)
  
  if(num == 0):
    print("No. Please try again\n")
  elif(num == 1):
    print("Wrong. Try once  more.\n")
  elif(num == 2):
    print("Don't give up!\n")
  elif(num == 3):
    print("No. Keep trying\n")
 
def giusto():
 num = random.randrange(0, 4)
  
 if(num == 0):
   print("Very Good!\n")
 elif(num == 1):
   print("Excellent!.\n")
 elif(num == 2):
   print("Nice work!\n")
 elif(num == 3):
   print("Keep up the good work!\n")
  
def estrai_num():
  return random.randrange(1, 10)

if __name__ == '__main__':
  giusto()
  sbagliato()
  print(estrai_num())
